app/serializers/api/v3/statements_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module API
+  module V3
+    class StatementsSerializer < Blueprinter::Base
```

Comment:
Im not sure we need this as we can pass a collection to a serialiser? see collections in https://github.com/procore-oss/blueprinter